server/glassfish-demo/README.md

```diff
@@ -0,0 +1,18 @@
+# Glass Fish
+
+This is an example blueprint with Glass fish.  This *blueprint* leverages xebialabs unsupported docker containers. To run this template you can use the docker-compose file ([doccker-compose.yml](docker-comnpose.yml)).  You will also need access to a JIRA instance.  This template creates an XL Deploy application, a Glass Fish environment and a release template to deploy the application to the sample environment.
```

Comment:
Why are GlassFish and JIRA in the same blueprint? Is there a logical link between the two?

Reply:
Many potential customers and new customers need and want some guidance on how various tools can be orchestrated with XL Release

Reply:
True. But does that warrant them being put in a blueprint? For a demo this would certainly make sense, but how many users do you expect to have a project that uses both GlassFish and JIRA?